src/vm/hints/squash_dict_utils.rs

```diff
@@ -0,0 +1,2485 @@
+use std::collections::HashMap;
+
+use num_bigint::BigInt;
+use num_traits::{FromPrimitive, ToPrimitive};
+
+use crate::{
+    bigint, bigintusize,
+    serde::deserialize_program::ApTracking,
+    types::{exec_scope::PyValueType, relocatable::MaybeRelocatable},
+    vm::{errors::vm_errors::VirtualMachineError, vm_core::VirtualMachine},
+};
+
+use super::{
+    dict_hint_utils::DICT_ACCESS_SIZE,
+    hint_utils::{
+        get_address_from_var_name, get_int_from_scope, get_list_from_scope, get_range_check_builtin,
+    },
+};
+
+fn get_access_indices(vm: &mut VirtualMachine) -> Option<HashMap<BigInt, Vec<BigInt>>> {
```

Comment:
I think we should be able to use `u64` for indices. We can't have a dict bigger than addressable memory.